pennylane_lightning_gpu/src/tests/Test_StateVectorCudaManaged_NonParam.cpp

```diff
@@ -828,3 +828,63 @@ TEMPLATE_TEST_CASE("StateVectorCudaManaged::applyCSWAP",
         }
     }
 }
+
+TEMPLATE_TEST_CASE("Sample", "[StateVectorCudaManaged_Nonparam]", float, double) {
+    constexpr uint32_t twos[] = {
+        1U << 0U,  1U << 1U,  1U << 2U,  1U << 3U,  1U << 4U,  1U << 5U,
+        1U << 6U,  1U << 7U,  1U << 8U,  1U << 9U,  1U << 10U, 1U << 11U,
+        1U << 12U, 1U << 13U, 1U << 14U, 1U << 15U, 1U << 16U, 1U << 17U,
+        1U << 18U, 1U << 19U, 1U << 20U, 1U << 21U, 1U << 22U, 1U << 23U,
+        1U << 24U, 1U << 25U, 1U << 26U, 1U << 27U, 1U << 28U, 1U << 29U,
+        1U << 30U, 1U << 31U};
+
+    // Defining the State Vector that will be measured.
+    size_t num_qubits = 3;
+    size_t data_size = std::pow(2, num_qubits);
+
+    std::vector<std::complex<T>> init_state(data_size, 0);
+    init_state[0] = 1;
+    SVDataGPU<TestType> svdat{num_qubits, init_state};
+
+    T alpha = 0.7;
+    T beta = 0.5;
+    T gamma = 0.2;
+    svdat.cuda_sv.applyOperations(
+        {"RX", "RY", "RX", "RY", "RX", "RY"}, {{0}, {0}, {1}, {1}, {2}, {2}},
+        {false, false, false, false, false, false},
+        {{alpha}, {alpha}, {beta}, {beta}, {gamma}, {gamma}});
+
+    vector<TestType> expected_probabilities = {0.687573, 0.013842, 0.089279,
```

Comment:
```suggestion
    std::vector<TestType> expected_probabilities = {0.687573, 0.013842, 0.089279,
```

Since we used `std` everywhere else, best to keep it consistent here.